File: Data Structures & Algorithms/two-integer-sum/submission-1.cpp
class Solution {
public:
    
    vector<int> twoSum(vector<int>& nums, int target) {
        int localbuffer;
        for(int i = 0; i<nums.size()-1; i++)
        {
            localbuffer = nums[i];
            vector<int> retnums = {0, 0};
            for (int j=i+1; j<nums.size(); j++)
            {
                if(localbuffer + nums[j] == target) return retnums = {i,j};
            }
        }
    }
};

File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Set<Integer> set = new HashSet<>(nums.length);
        for (int x: nums)
        {
            if (!set.add(x))
            {
                return true;
            }
        }
        return false;
    }
}